source/includes/_investors.md

```diff
@@ -2015,97 +2015,149 @@ Any existing ISA data is preserved.
+    "companyName": "Fake company"
+  },
+  "address": {
+    "country": "GBR",
+    "lineOne": "1 The High Street",
+    "lineTwo": "",
+    "lineThree": "",
+    "postcode": "AA1 1AA",
+    "townCity": "London",
+    "region": "Greater London"
+  },
+  "accountTypes": [
+    "GOJI_INVESTMENT",
+  ],
+  "entityType": "CORPORATE",
+  "nationalities": [
+    {"nationality": "GB"},
+    {"nationality": "CH"}
+  ],
+  "dateOfBirth": "1990-01-19",
+  "migrationDetails": {
+    "existingClientId": "existingClientId",
+    "diligenceMigrationOption": "FULL_CHECK",
+    "previousCheckDate": "2000-01-23"
+  },
+  "investorDeclarationType": "HIGH_NET_WORTH",
+  "contactDetails": {
+    "emailAddress": "example@example.com",
+    "telephoneNumber": "07123456789"
+  }
+}
 ```
 
 ```http 
 HTTP/1.1 200 OK
 Content-Type: application/json
 
 {
-  "lastName" : "lastName",
-  "corporateDetails" : {
-    "companyType" : "companyType",
-    "registrationNumber" : "registrationNumber",
-    "companyName" : "companyName"
+  "title": "MS",
+  "firstName": "Jane",
+  "lastName": "Doe",
+  "corporateDetails": {
+    "companyType": "LLC",
+    "registrationNumber": "AB123456",
+    "companyName": "Fake company"
   },
-  "clientId" : "clientId",
-  "address" : {
-    "country" : "country",
-    "lineTwo" : "lineTwo",
-    "townCity" : "townCity",
-    "postcode" : "postcode",
-    "lineOne" : "lineOne",
-    "lineThree" : "lineThree",
-    "region" : "region"
+  "address": {
+    "country": "GBR",
+    "lineOne": "1 The High Street",
+    "lineTwo": "",
+    "lineThree": "",
+    "postcode": "AA1 1AA",
+    "townCity": "London",
+    "region": "Greater London"
   },
-  "entityType" : "INDIVIDUAL",
-  "employmentDetails" : {
-    "jobTitle" : "jobTitle",
-    "employmentStatus" : "EMPLOYED_FULL_TIME"
-  },
-  "dateOfBirth" : "2000-01-23",
-  "title" : "MISS",
-  "contactDetails" : {
-    "emailAddress" : "emailAddress",
-    "telephoneNumber" : "telephoneNumber"
+  "accountTypes": [
+    "GOJI_INVESTMENT",
+  ],
+  "entityType": "CORPORATE",
+  "nationalities": [
+    {"nationality": "GB"},
+    {"nationality": "CH"}
+  ],
+  "dateOfBirth": "1990-01-19",
+  "migrationDetails": {
+    "existingClientId": "existingClientId",
+    "diligenceMigrationOption": "FULL_CHECK",
+    "previousCheckDate": "2000-01-23"
   },
-  "firstName" : "firstName",
-  "nationalInsuranceNumber" : "nationalInsuranceNumber",
-  "accountTypes" : "GOJI_INVESTMENT",
-  "investmentDeclarationType" : "RESTRICTED"
+  "investorDeclarationType": "HIGH_NET_WORTH",
+  "contactDetails": {
+    "emailAddress": "example@example.com",
+    "telephoneNumber": "07123456789"
+  }
 }
 ```
 ### Description
 Migrates an existing investor onto the Goji system.
+
+<aside class="notice">
+The required <code>migrationDetails</code> allows the platform to provide an existing investor client ID where applicable and provide the diligence status as agreed with Goji before migration takes place. 
+</aside>
```

Comment:
I don't understand this.
I've copied this from the description above but why does it say `existing investor ID where applicable` if it's mandatory?

Reply:
![image](https://user-images.githubusercontent.com/3105832/84821419-02704200-b013-11ea-90cb-1fb3b274155f.png)

![image](https://user-images.githubusercontent.com/3105832/84821490-19169900-b013-11ea-9a31-4635e6cd9107.png)

But still the  `migrationDetails.existingClientId` can be empty.
This is so weird.
![image](https://user-images.githubusercontent.com/3105832/84821559-38adc180-b013-11ea-82b5-76ce5a4b0db9.png)